Update gettingstarted.md (#212)
codatio/codat-docs: docs/sync-for-expenses/gettingstarted.md

- One will be to their accounting platform
- The other will be to the partner expense integration

> When you create the partner expense connection it will already be 'Linked' so you won't need to authorise this connection again. 

Reviewer: ```suggestion
When you create the partner expense connection it will already be 'Linked' so you won't need to authorize this connection again. 
```

> When you create the partner expense connection it will already be 'Linked' so you won't need to authorise this connection again. 

Reviewer 2: ```suggestion
The partner expense connection is created with the 'Linked' status, so you won't need to authorize this connection again. 
```

Author: Changed made manually
